### DISCUSSION TITLE
Creating a coding profiles stats generator

### DISCUSSION THREAD

**Discussion by vivekanand-vr** [anuraghazra/github-readme-stats#3756] [Ideas]
- As we all have solved a number of problems on leetcode and participated in a lot of contests 
- I believe displaying our ranks/problems solved also adds significant value to out portfolio
- If possible we can create a similar way of getting coding profiles

**Comment by juan-miii** [comment]
Statistics from outside platforms sounds really interesting, but probably does not match GitHub stats. I include here the links to some repos that actually do that (none of them are related to me), but I believe you will find them useful. [KnlnKS/leetcode-stats](https://github.com/KnlnKS/leetcode-stats), [JacobLinCool/LeetCode-Stats-Card](https://github.com/JacobLinCool/LeetCode-Stats-Card), [JeremyTsaii/leetcode-stats-api](https://github.com/JeremyTsaii/leetcode-stats-api). Hope this helps!

**Comment by vivekanand-vr** [comment]
Thanks @juan-miii
